### ISSUE TITLE
JS changes aria-expanded value on the wrong element for navbar

### ISSUE THREAD

**Issue by andersonjeccel** [mautic/mautic#12935]
### Mautic Version

5.0.x series

### PHP version

8.0.30

### What browsers are you seeing the problem on?

Microsoft Edge

### What happened?

# Description
The main interface navigation bar has an Accessibility issue where the JavaScript code used to manipulate menu attributes is adding the "aria-expanded" attribute to the list instead of the link that triggers the list's expansion. 
This causes screen readers to be unable to detect when a navigation bar menu is open.

### // Mautic Components menu

```
<nav>
<li>
<a href="javascript:void(0);" data-target="#mautic_components_root_child" data-toggle="submenu" data-parent=".nav" data-menu-link="mautic_components_root" id="mautic_components_root" role="menuitem" aria-haspopup="true" aria-label="Components">
<span><span><span>
</a>
```

### // When opened, nothing happens to `<a>`, but the `<ul>` submenu changes from this:

```
<ul class="nav-submenu collapse" id="mautic_components_root_child">
<li>
<a>
<span>
```

### // To this:

`<ul class="nav-submenu collapse in" id="mautic_components_root_child" aria-expanded="true" style="">`

## What should happen

The `<a>` should be modified to receive the aria-expanded attribute.

```
<nav>
<li>
<a href="javascript:void(0);" data-target="#mautic_components_root_child" data-toggle="submenu" data-parent=".nav" data-menu-link="mautic_components_root" id="mautic_components_root" role="menuitem" aria-haspopup="true" aria-label="Components" aria-expanded="true">
```

The solution involves modifying the JavaScript code to directly manipulate the link `<a>` element instead of the `<ul>` list element when changing the open/closed state, and vice versa.

### How can we reproduce this issue?

Step 1: Open Mautic dashboard
Step 2: Open Dev Tools and find the element
Step 3: Click on the Components to open and see the HTML changes happening

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by RCheesley** [comment]
@andersonjeccel is this one addressed now with the new UI changes?

**Comment by andersonjeccel** [comment]
@RCheesley nope
